c_src/mac/dext.cpp

```diff
@@ -23,7 +23,9 @@ int init_sink() {
     /**/
     client->connected.connect([copy] {
                                   std::cout << "connected" << std::endl;
-                                  copy->async_virtual_hid_keyboard_initialize(pqrs::hid::country_code::us);
+                                  pqrs::karabiner::driverkit::virtual_hid_device_service::virtual_hid_keyboard_parameters parameters;
+                                  parameters.set_country_code(pqrs::hid::country_code::us);
+                                  copy->async_virtual_hid_keyboard_initialize(parameters);
                               });
     client->connect_failed.connect([](auto&& error_code) {
                                        std::cout << "connect_failed " << error_code << std::endl;
```

Comment:
Could you also add something like:
```diff
+                                       std::cout << "You may need to start Karabiner " << std::endl;
```
since it isn't started automatically anymore and this would be a very bad error message then.